IMDB Watchlist importer

lechat opened the pull request:
Added ability to watch for IMDB Watchlist (Issue #146) The code is based on Trakt watchlist importer.

It operates using URL of the "Export this list" link, which is displayed at the bottom of each list page. I added configuration section under "Extras" tab for it.

lechat committed:
IMDB Watchlist importer (Issue #146)

Commit:
Fixed userscript so it will work from IMDB pages opened fron trailers
(issues #398, #400)

RuudBurger commented on app/views/config/index.html:
```diff
@@ -443,6 +443,14 @@
+				<div>Why isn't it in the normal Appstore you ask? Because Apple is being a bitch!</div>
+			</fieldset>
+		</div>
+
```
Why is this?

RuudBurger commented:
Looking good, but you need to add the enabled to https://github.com/RuudBurger/CouchPotato/blob/master/app/controllers/config.py#L63
And there is some code that isn't needed (see other comment)

lechat committed:
Removed unnecessary section from config/inde and added
IMDBWatchlist.enabled into config

lechat commented:
Yes, the extra section was unnecessary and was a part of my try outs. Thanks for noticing it! Code review FTW! :) 

I removed it and added missing bit.

RuudBurger commented:
Thanks for your work. WIll use this in the new build too, thanks!
